visualize/static/scripts/visualizations/new_view.js

```diff
@@ -0,0 +1,106 @@
+"use strict";
+
+var NewView = DefaultView.extend({
```

Comment:
I'll get rid of it. It was Ethan's practice visualization, and I like to
use it for an example, but I think it's on slack now.

On Wed, Feb 10, 2016 at 9:25 AM, Connor Cook <notifications@github.com>
wrote:

> In visualize/static/scripts/visualizations/new_view.js
> <https://github.com/BYU-NLP-Lab/topicalguide/pull/87#discussion_r52482132>
> :
>
> > @@ -0,0 +1,106 @@
> > +"use strict";
> > +
> > +var NewView = DefaultView.extend({
>
> What is this view for? Is it just a test, "hello world" type thing? If so,
> it probably doesn't need to be pushed up (unless we want to use it to show
> people how to create visualizations?)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BYU-NLP-Lab/topicalguide/pull/87/files#r52482132>.
>